docs/redenomination.md

```diff
@@ -0,0 +1,123 @@
+Plancks to 10e10 Plancks. In other words, the DOT went from having twelve decimal places of
+precision to only ten decimal places of precision. The overall effect of this change was that the
+amount of Polkadot's smallest unit, the Planck remained constant, while DOT balances for all holders
+was increased by a factor of ten.
+
+```
+   Before the change the decimal was here
+   v
+  1.000000000000 DOT
+
+  100.0000000000 DOT
+     ^
+     After the change the decimal is here
+```
+
+## Origins
+
+The initial vote for redenomination occurred as a [referendum][referendum 52] on the Kusama
+blockchain. The referndum was summarized as having four effects if approved by KSM holders.
+
+> - The total allocations of DOTs will increase one hundred times from 10 million to 1 billion.
+> - DOT allocation balances will increase by a factor of one hundred, such that 1 DOT will be 100
+>   DOTs.
+> - The distribution of DOTs does not change, and holders of DOTs still own an equal share of the
+>   network as before the change.
+> - The precision of DOT will change from 12 decimal places to 10 decimal places.
+> - The main benefit of this change is to avoid using small decimals when dealing with DOT and to
+>   achieve an easier calculation system.
+
+The initial referendum was proposed prior to the Polkadot genesis block under the assumption that
+making a redenomination would be simpler before the Polkadot chain was live. However, many in the
+community pointed out the disconnect of the two networks and how it was unfair for holders of DOT to
+be impacted by a vote by a different token holder set. For this reason, the Web3 Foundation [decided
+to make a new vote on Polkadot][blog 1] when it went live, although the Kusama vote ended with a
+majority in favor of the redenomination change.
+
+A bolded quote from the Foundation read:
+
+> However, given the non-negligible amount of opposition, including from some within the ranks of
+> Web3 Foundation and Parity, **the Foundation decided that we cannot, in good faith, sponsor
+> redenomination at present.**
+
+## The Vote
+
+After the genesis block of Polkadot was created and the network was running with a decentralized
+community of validators securing the network, it was decided to put the redenomination topic up to a
+vote again. This time, the vote was explicitly binding - whereas the vote on Kusama was of course
+non-binding being at best a way to capture of a signal and not able to affect the Polkadot chain
+directly.
+
+Due to the feedback received during the Kusama referendum, the Polkadot vote was to be held in an
+approval vote style, with four available options. DOT holders could issue votes for any
+configuration of the four options: no change, a change of 10x, a change of 100x, or a change of
+1000x. The vote logic was contained in a specially-built Substrate pallet that was included in
+Polkadot's runtime for this specific purpose.
+
+The important notes of the Polkadot vote are as follows:
```

Comment:
```suggestion
The important notes of a Polkadot vote are as follows:
```

Reply:
In this case, I'm referring to a specific vote on Polkadot so it should be "the Polkadot vote".